### PR TITLE
feat: addition of pm authentication processor new flow

### PR DESCRIPTION
**Pull request by gitanjli525**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
PM Authentication Processor new flow

## Where to test it?
- [ ] INTEG
- [ ] SANDBOX
- [ ] PROD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Review by JeevaRamu0104 — DISMISSED** [review]

File: src/screens/PMAuthenticationProcessor/PMAuthenticationTableEntity.res [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  | ProfileId
+  | ProfileName
+  | ConnectorLabel
+
+let defaultColumns = [Name, ProfileId, ProfileName, ConnectorLabel, Status, Disabled, TestMode]
+
+let getHeading = colType => {
+  switch colType {
+  | Name => Table.makeHeaderInfo(~key="connector_name", ~title="Processor", ~showSort=false)
```
Comment (PritishBudhiraja) [inline src/screens/PMAuthenticationProcessor/PMAuthenticationTableEntity.res:17]: Can we remove showSort as @sagarnaikjuspay was removing it

Comment (gitanjli525) [reply src/screens/PMAuthenticationProcessor/PMAuthenticationTableEntity.res:17]: that PR is not merged

Comment (PritishBudhiraja) [reply src/screens/PMAuthenticationProcessor/PMAuthenticationTableEntity.res:17]: okay as sagar to remove from this place if this gets merged first

**Review by PritishBudhiraja — CHANGES_REQUESTED** [review]
Like why we are following backend naming convention throughout codebase. In FE, JS standard is camelCase try to change this to that not snake_case

**Comment by gitanjli525** [comment]
> Like why we are following backend naming convention throughout codebase. In FE, JS standard is camelCase try to change this to that not snake_case

We can make that change in a separate PR, here it is done so as to maintain consistency with existing code itself.

**Review by PritishBudhiraja — APPROVED** [review]

**Review by PritishBudhiraja — APPROVED** [review]

**Review by sagarnaikjuspay — APPROVED** [review]

**Pull request merged**